### ISSUE TITLE
Make search results more user-friendly

### ISSUE THREAD

**Issue by meiqimichelle** [18F/hub#160]
Right now, if I run a search, I get what look like a combination of url slugs and page titles.
![screenshot 2015-02-13 09 32 30](https://cloud.githubusercontent.com/assets/4827522/6191000/7089fb08-b363-11e4-8168-3fa8bc7ba615.png)

Suggest having search return...I'm not sure what. Maybe a page's h1 ?? Whatever approach we decide on, the search returns should be standardized.

Discuss.

**Comment by meiqimichelle** [comment]
....and then by extension, the search could be weighted for h1. ie, right now the `how to` page often gets listed first, but likely the one right below it....the `answer` to the `question` on the `how to` is what people really want.

**Comment by mbland** [comment]
Yeah, I noticed that... :-) It seems we should guarantee, at the very least, that each page has its `title` tag set, as that appears to be what the search engine uses when present.

**Comment by meiqimichelle** [comment]
Ah. In that case we just need to change the template, I think -- mustachify the `<title>` tag.
